mods/rv/rules/ai.yaml

```diff
@@ -957,8 +973,8 @@ Player:
 			gapowr: 16
 			napowr: 8
 			nanrct: 1
-			yapowr: 4
-			yapowr-full: 8
+			yapowr: 8
+			yapowr-full: 10
```

Comment:
You used 12 everywhere else, why is this 10?

Reply:
oh, let me change that